### DISCUSSION TITLE
DNG conversion bug

### DISCUSSION THREAD

**Discussion by FrontierDK** [ImageMagick/ImageMagick#3066] [Development]
Just installed ImageMagick-7.0.10-54-Q16-HDRI-x86-dll.exe (deleted ImageMagick-7.0.10-53-Q16-HDRI-x86-dll.exe), and now, I can't convert DNG images.

I get this bug:
![bug](https://user-images.githubusercontent.com/25281742/103469749-04101600-4d69-11eb-9e56-451b84857101.png)

Deleting the new version and reinstalling ImageMagick-7.0.10-53-Q16-HDRI-x86-dll.exe fixed the issue.

**Comment by fmw42** [comment]
IM 7.0.10.54 works fine for me on my Mac to convert your DNG file.

Edit your policy.xml file to permit larger images. Your default values are not allowing such large files either by dimension or file size. You probably had higher permissions before.

**Comment by FrontierDK** [comment]
I have never touched that file before...

At any rate, I uncommented the 2 lines mentioning height and width, adjusted the numbers to VERY high settings...no difference.
```
  <policy domain="resource" name="width" value="50MP"/>
  <policy domain="resource" name="height" value="50MP"/>
```
Uninstalling ImageMagick-7.0.10-**54**-Q16-HDRI-x86-dll.exe removed the entire program folder. Installed ImageMagick-7.0.10-**53**-Q16-HDRI-x86-dll.exe ...and encoding works again.

**Reply by fmw42** [reply]
In addition, your area resource needs to be more than 40323024 pixels for this image.

**Comment by dlemstra** [comment]
Can you share your input image? And can you also reproduce this with the 64bit version of ImageMagick?

**Reply by FrontierDK** [reply]
Original DNG is at http://mitip.dk/Test-original.DNG

I just tried with the x64 version - and it works in that version. However, that version can't be used with classic ASP, along with other 32bit components. Installing the 32bit version, version 7.0.10-53 fixes the 32-bit issue. I did a bit-by-bit comparison of policy.xml (before editing) to the working 32-bit version, and they're identical. So the issue isn't in that file.

Changing/uncommenting the area to 10GB still doesn't remove the stop warning about width/height.

**Comment by urban-warrior** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in the GIT master branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ https://www.imagemagick.org/download/beta/ by sometime tomorrow.

**Comment by FrontierDK** [comment]
Just tried with the new ImageMagick 7.0.10-55 32bit, and I can confirm that DNG > JPEG conversion is working now :)
